docs/working-with-cdk-for-terraform/terraform-functions.md

```diff
@@ -0,0 +1,27 @@
+# Terraform Functions
+
+Terraform provides a set of functions that can be used through the CDK for Terraform.
+
+```ts
+import { Fn } from "cdktf";
+
+new vpc() = new VPC(this, "vpc", {});
+new LoadBalancer(this, "lb", {
+  name: "main-lb",
+  subnet: Fn.Network.cidrsubnet(
```

Comment:
Looks like namespacing got dropped. Would be nice to have though.

Reply:
It might work when following this https://github.com/aws/jsii/blob/ff03310e1540ddf5364ab20390e854be9b57bc17/gh-pages/content/specification/2-type-system.md#declaration